### PR TITLE
Shift website deployment to github pages

### LINKED ISSUE #1340: [NG] Testing and Tracking the move to Github pages
**Task**

This issue is to be used as a tracking point for the move to Github pages. The aim is to build the site, test it locally, report back any roadblocks and fix them before the official move.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by shubhank-saxena**
**Description**

This PR fixes #1340 

**Notes for Reviewers**
There are some steps that are to be followed -
- As soon as PR is merged, the following command needs to be run from layer5-ng branch -  ```npm run deploy```. This will create a branch called gh-pages.
- After this, open up settings and you will that the Github automatically picks up gh-pages as a branch to be deployed from. 
- Also add the custom domain to the field below for re-routing. 
- This will involve setting up of A name field in the DNS entries of the domain
- This is how the Github and DNS config should look like (this is an example one which I used for deployment)

2021-01-05_11-55-30_screenshot

2021-01-05_11-55-47_screenshot
 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Good work. Thank you, @shubhank-saxena. 👍

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,3 +24,21 @@ jobs:
...
+
+
+name: layer5-ng deploy
+on:
+  push:
+    branches: [layer5-ng]
+jobs:
+  deploy-GH:
+    name: Deploy on GH pages
+    runs-on: ubuntu-latest
```
Comment (hwpplayer1) [inline .github/workflows/ci.yml:36]: Can we add an information here "Ubuntu Latest" ?

Comment (leecalcote) [reply .github/workflows/ci.yml:36]: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md

**Comment by leecalcote** [comment]
This is the week, @shubhank-saxena. 😄

**Pull request closed without merging**